test/index.js

```diff
@@ -0,0 +1,41 @@
+var R = require('..');
+var eq = require('./shared/eq');
+var fs = require('fs');
+var path = require('path');
+
+
+var sourceMethods = function(dir) {
+  var isJsFile = function(file) { return file.match(/\.js$/); }
+  var removeJsEnding = function(file) { return file.replace('.js', ''); }
+  return fs.readdirSync(dir).filter(isJsFile).map(removeJsEnding);
+}
+
+/**
+ * Convention is
+ *  * Actual API—all `./src/*.js` files are top level API methods
+ *  * Exported API—object in `./index.js` to be exported
+ *  * Actual and exported APIs should be the same
```

Comment:
This paragraph decided is a way to get everyone know about convention we agreed on this thread